Formula/gobject-introspection.rb

```diff
@@ -14,10 +14,7 @@ class GobjectIntrospection < Formula
   depends_on "glib"
   depends_on "cairo"
   depends_on "libffi"
-  # System python in Mavericks or below has bug in distutils/sysconfig.py, which breaks the install.
-  #    Caught exception: <type 'exceptions.AttributeError'> AttributeError("'NoneType' object has no attribute 'get'",)
-  #    > /System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/distutils/sysconfig.py(171)customize_compiler()
-  depends_on "python" if MacOS.version <= :mavericks
+  depends_on "python" # never switch back to system python!
```

Comment:
You're forgetting that a lot of people use pyenv, which installs a non-universal python which shouldn't cause any problems, correct? If so, then there should at least be a build option to use the current python (or perhaps the path to a specific python binary).